libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/Notification.java

```diff
@@ -29,10 +29,17 @@
     // key value pair describing additional details about notification
     private Map<String, String> details;
 
-    public Notification (String type, Set<String> recipients, Map<String, String> details) {
+    // Utility class to convert the notification into an email
+    private NotificationToEmailConverter notificationToEmailConverter;
+
+    public Notification (String type,
```

Comment:
NotificationToEmailConverter will be implemented inside each NotificationTask. Its purpose is to convert the notification into an email